src/blockcholesky.jl

```diff
@@ -0,0 +1,27 @@
+
+# Function for cholesky decomposition on block matries
+# 'cholesky' is the build-in one in LAPACK
+
+function Bcholesky(A::Symmetric{<:Any,<:BlockArray})
```

Comment:
In src/BlockArrays.jl, add
```julia
import LinearAlgebra: cholesky, cholesky!
```
And then you can overload `cholesky` and `cholesky!`